.github/workflows/datafuse-release.yml

```diff
@@ -67,3 +67,41 @@ jobs:
           file_glob: true
           overwrite: true
           body: ${{ steps.changelog_reader.outputs.log_entry }}
+  release-docker:
+    name: Release assets
+    runs-on: ${{ matrix.config.os }}
+    needs: [publish]
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+          # Linux
+          - { os: ubuntu-latest, toolchain: stable, target: x86_64-unknown-linux-gnu, cross: false }
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Get tag name
+        id: tag_name
+        run: |
+          echo ::set-output name=current_version::${GITHUB_REF#refs/tags/v}
+        shell: bash
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Build and push
+        id: docker_build
+        uses: docker/build-push-action@v2
+        with:
+          push: true
+          tags: ${{ secrets.DOCKERHUB_NAMESPACE }}/datafuse:${{ steps.tag_name.outputs.VERSION }}
```

Comment:
may not get `steps.tag_name.outputs.VERSION` now

Reply:
![IMG_20210810_142338](https://user-images.githubusercontent.com/36896360/128818342-43bd30fa-99e3-41f0-b103-246b77918739.jpg)

Reply:
I think while publish job finished(https://github.com/datafuselabs/datafuse/issues/1397), docker start to build, thus this should work

Reply:
strange, let me adopt some other methods

Reply:
https://github.community/t/how-to-get-just-the-tag-name/16241/7 adopted this trick

Reply:
@ZhiHanZ line 41 also need fix

Reply:
https://github.com/ZhiHanZ/hello-github-actions/runs/3288135110?check_suite_focus=true trick works, I think install.sh scripts should be changed accordingly